#!/usr/bin/env python
PACKAGE = "navex_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("last_point",   bool_t,   0, "Check the distance between the goal and the last point on path",  False)

exit(gen.generate(PACKAGE, "navex_navigation", "GoalTrajectoryMatcher"))